src/conditionalLogicService/evaluateConditionalOptionsPredicate.ts

```diff
@@ -11,15 +11,16 @@ export default function evaluateConditionalOptionsPredicate({
 }): boolean {
   return predicate.optionIds.some((optionId) => {
     const option = predicateElement.options?.find((o) => o.id === optionId)
-    if (option) {
+    const optionValue = option?.value ?? optionId
```

Comment:
Can we please add a comment here to outline the use case for an option not being found in the predicate element but the `optionId` being the same as a `value` of another option.